acme/acme/magic_typing.py

```diff
@@ -6,12 +6,13 @@
 """
 import warnings
 from typing import *  # pylint: disable=wildcard-import, unused-wildcard-import
-from typing import Collection, IO
+from typing import Any
```

Comment:
No it was not. Thanks!

Reply:
Turned out it was necessary in fact, because of how the test `test_import_success` is `magic_typing_test` is designed. Given that this module is deprecated anyway, it do not think it is a big deal to just keep the redundant import for test purposes.